geemusic/utils/music.py

```diff
@@ -68,6 +68,18 @@ def get_stream_url(self, song_id):
     def get_all_user_playlist_contents(self):
         return self._api.get_all_user_playlist_contents()
 
+    def rate_song(self, song, rating):
+        try:
```

Comment:
Why use a `try`/`catch` here/what's getting thrown that prevents the rating?

It seems like looping through every track in the user's library is a heavy operation that we might want to avoid unless we absolutely have to go down that route.